app/src/editor/canvas/components/Resizable/SizeConstraintProvider/Probe/index.jsx

```diff
@@ -0,0 +1,51 @@
+// @flow
+
+import React, { useRef, useEffect, useContext } from 'react'
+import { type Ref } from '$shared/flowtype/common-types'
+import { Context as SizeConstaintContext } from '..'
+import styles from './probe.pcss'
+
+type Props = {
+    group: string,
+    height?: 'auto' | number,
+    id?: string,
+    width?: 'auto' | number,
+}
+
+const Probe = ({ group, id: idProp, width, height }: Props) => {
+    const id = idProp || group
+    const ref: Ref<HTMLDivElement> = useRef(null)
+    const { setWidth, setHeight, probeRefreshCount } = useContext(SizeConstaintContext)
+
+    useEffect(() => {
+        const { current } = ref
+
+        if (!current) {
+            return
+        }
+
+        const { width: w, height: h } = current.getBoundingClientRect()
```

Comment:
Not sure if it's an issue here. That `useEffect` runs rarely.
- Resizing a module does \*not\* trigger it
- Probe's props usually don't change (never, atm)

It gets called mostly because of the `probeRefreshCount` (requested recalculation). It doesn't happen often (depends on changes made by the user), and it's a single call per request.

Reply:
👍